packages/generate-html/test/generate-html.spec.ts

```diff
@@ -73,19 +87,52 @@ describe('GenerateHTML', function () {
     ignoreDuplicateAttributes: false,
   };
 
+  // eslint-disable-next-line sonarjs/cognitive-complexity
   testsMap.forEach((testSuite) => {
     const {
       certificatePath,
       expectedHtmlFromHbs,
       expectedHtmlFromMjml,
+      localOnly,
+      localTemplatePath,
+      localTemplatePartialPath,
       schemaExtraTranslationsPath,
       schemaTranslationsPath,
       type,
       version,
     } = testSuite;
     describe(`For ${type} - version ${version}`, () => {
+      let certificate: Record<string, unknown>;
+      let translations: Translations | undefined;
+      let extraTranslations: ExtraTranslations | undefined;
+      let templatePartial: HandlebarsTemplateDelegate<any> | undefined;
+      let localOnlyOptions: GenerateHtmlOptions | undefined;
+
+      beforeAll(() => {
+        certificate = JSON.parse(readFileSync(certificatePath, 'utf8'));
+        translations = JSON.parse(readFileSync(schemaTranslationsPath, 'utf8'));
+        extraTranslations = schemaExtraTranslationsPath
+          ? JSON.parse(readFileSync(schemaExtraTranslationsPath, 'utf8'))
+          : {};
+
+        if (localTemplatePartialPath) {
+          const partialString = readFileSync(localTemplatePartialPath).toString();
+          templatePartial = compile(partialString);
+        }
+
```

Comment:
We could have multiple `templatePartialPaths` so you would need to iterate over each of them to load the corresponding file.